server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderIT.java

```diff
@@ -186,7 +186,7 @@ public void testRestoreSnapshotAllocationDoesNotExceedWatermark() throws Excepti
         );
 
         // increase disk size of node 0 to allow just enough room for one shard, and check that it's rebalanced back
-        getTestFileStore(dataNodeName).setTotalSpace(smallestShard.size + WATERMARK_BYTES + 1L);
+        getTestFileStore(dataNodeName).setTotalSpace(smallestShard.size + WATERMARK_BYTES);
```

Comment:
`Expected: Expected to contain exactly one value from [<[txfuhzdlsd][0]>] but: was <[[txfuhzdlsd][5]]>`.
According to the build logs:
```
[2023-02-14T05:03:10,802][INFO ][o.e.c.r.a.d.DiskThresholdDeciderIT] [testRestoreSnapshotAllocationDoesNotExceedWatermark] Created shards with sizes {
[txfuhzdlsd][4]=57045, 
[txfuhzdlsd][5]=55594, <----
[txfuhzdlsd][2]=60575, 
[txfuhzdlsd][3]=57818, 
[txfuhzdlsd][0]=55593, <----
[txfuhzdlsd][1]=58250
}
```

I guess we do not need that +1 in test

Reply:
The repro line only affected the index name, but not the actual shard size so it is tricky to reproduce.

This line have not changed size the test was introduced in 2020: https://github.com/elastic/elasticsearch/pull/60460
I am surprised we have not noticed this before.